index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Random Word Pick</title>
+</head>
+<body>
+    <div class="adminCommandsWrapper">
+        <button id="submitModeButton" onclick="buttonPressed(0);" class="clearButton">Submit mode</button>
+        <button id="clearButton" onclick="buttonPressed(1);" class="clearButton">Clear</button>
+        <input type="password" onblur="clearWordsBarBlur();" placeholder="password" id="clearWordInput" class="clearWordBar">
+    </div>
+  
+    <h1>Random Word Picker</h1>
+  
+    <div class="buttonWrapper">
+        <button onclick="pickWord();" class="bigButton">Pick Word</button>
```

Comment:
You want use a more descriptive class identifier rather than "bigButton" since this class is only being used for this one specific use case -- perhaps something simple like newWordButton